src/main/java/mate/academy/springboot/datajpa/service/impl/ProductServiceImpl.java

```
@@ -0,0 +1,40 @@
+package mate.academy.springboot.datajpa.service.impl;
+
+import java.math.BigDecimal;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.datajpa.model.Product;
+import mate.academy.springboot.datajpa.repository.ProductRepository;
+import mate.academy.springboot.datajpa.service.ProductService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ProductServiceImpl implements ProductService {
+    private final ProductRepository productRepository;
+
+    @Override
+    public Product save(Product product) {
+        return productRepository.save(product);
+    }
+
+    @Override
+    public Product findById(Long id) {
+        return productRepository.findById(id).get();
```

let's throw an exception in case products isn't found